src/index.ts

```diff
@@ -24,7 +24,9 @@ const hasIgnoreClass = (e: any, ignoreClass: string): boolean => {
   let el = e.target;
 
   while (el) {
-    if (el?.classList.contains(ignoreClass)) return true;
+    // el.classList != null checks both for null and
+    // undefined since it's not a strict comparison
+    if (el && el.classList != null && el.classList.contains(ignoreClass)) return true;
```

Comment:
@vardani Thank you for this PR. Can we simplify the code by using the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to handle it?

Reply:
Sure thing, I'll go ahead and update it. Thanks!